etc/profile-m-z/qutebrowser.profile

```diff
@@ -18,6 +18,8 @@ include allow-bin-sh.inc
 include allow-python2.inc
 include allow-python3.inc
 
+ignore noexec ${HOME}
+
```

Comment:
The profile has apparmor?

Reply:
Yes indeed, qutebrowser.profile has apparmor. Wasn't sure how OP will test what I suggested [here](https://github.com/netblue30/firejail/issues/5639#issuecomment-1419285365). But you're correct, we need to take out apparmor too probably.